### DISCUSSION TITLE
Possibility to set dynamic min, max for integer fields

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2986] [Feature Requests]
I was wondering if there is any way we can have dynamic min, max for integer fields? Maybe even defined on another collection and link them somehow so that when we change in the base collection this min and max are also adjusted?

Or for specific entries, we have one min-max, for other different based on user, as an example.

**Comment by benhaynes** [comment]
The "Numeric" interface can set min/max per field... there's nothing dynamic yet, though you could use event hooks to set these.

**Reply by botzill** [reply]
yes, I did use that actually to change the `directus_fields` `options` to `maxValues` that I require.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
